yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -3,14 +3,18 @@
 #pragma once
 
 #include <cassert>
+#include <mutex>
 #include <utility>
 
 #include "yarpl/flowable/Flowable.h"
 #include "yarpl/flowable/Subscriber.h"
 #include "yarpl/flowable/Subscription.h"
 #include "yarpl/utils/credits.h"
 
+#include <boost/intrusive/list.hpp>
 #include <folly/Executor.h>
+#include <folly/SpinLock.h>
```

Comment:
do we use this?

Reply:
Oops, that's dead. I was testing if folly::Synchronized access was any faster with a spinlock.